src/components/EdgeMedia/EdgeMedia.tsx

```diff
@@ -97,6 +99,7 @@ export function EdgeMedia({
           muted={muted}
           html5Controls={html5Controls}
           onMutedChange={onMutedChange}
+          ref={videoRef}
```

Comment:
I think we can use `mediaRef` here, we just need to update the types I think 🤔 if it's a pita, then just forget it